### PR TITLE
Fixes #364: Fix Travis CI

### LINKED ISSUE #364: Travis Build Failing
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**
Travis build is failing

**Expected behavior:**
Travis build should not fail

### PR DESCRIPTION
**Pull request by S2606**
Fixes #364 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by S2606** [comment]
I am a bit confused if test_csv_upload is to be kept or not? Please tell me

**Comment by jajodiaraghav** [comment]
@S2606 Don't you want the csv upload to be tested?

**Comment by S2606** [comment]
@jajodiaraghav I think we dont need this due to disable generate badges button thing

**Comment by S2606** [comment]
Instead some test of same context but in other way can be written.

**Comment by vaibhavsingh97** [comment]
@S2606 Atleast have asked before sending the PR. If I haven't responded than you are free to send PR

**Comment by harshit98** [comment]
Hi @S2606 

Please change your commit message and PR title to -> `Fixes #(issue no.): title of fix`

**Comment by S2606** [comment]
@harshit98 Done

File: app/tests/test.py [hunk 1]
```diff
@@ -30,40 +30,39 @@ def test_menu_click_on_off(self):
...
         self.driver.find_element_by_name("file").send_keys(CSVpath)
         self.driver.find_element_by_name("image").send_keys(Imagepath)
-        self.driver.find_element_by_css_selector("form .btn-primary").click()
+        self.driver.find_element_by_css_selector("form .submit-btn").click()
         time.sleep(3)
         success = self.driver.find_element_by_css_selector(".flash-success")
-        self.assertIn(u'Your badges have been successfully generated!', success.text)
+        self.assertIn(u'Your badges have been created successfully.', success.text)
 
+    '''
```
Comment (vaibhavsingh97) [inline app/tests/test.py]: @S2606 Why it's commented?

**Comment by vaibhavsingh97** [comment]
@S2606 What's the status? and please tell me why the function `def test_csv_upload(self) ` is commented out?

**Comment by S2606** [comment]
@vaibhavsingh97 The function ``def test_csv_upload(self)`` was used to check if there comes a condition where user uploaded csv file but not PNG file for upload, an error would be thrown, but in as per this PR #358 The button will be enabled if and only if both csv and png file are uploaded.So this is the reason why i commented it out so that a new function similar to this can be created if it is needed.

And regarding conflicts will fix it in a day as i have exams going on.

**Comment by S2606** [comment]
@harshit98 Merge conflicts has been resolved..

**Comment by S2606** [comment]
@vaibhavsingh97 Done.

**Comment by mariobehling** [comment]
Thanks! Please iterate further from here. Good job.

**Pull request merged**